CMakeLists.txt

```diff
@@ -22,6 +22,16 @@ option(SOURCEMETA_CORE_CONTRIB_GOOGLEBENCHMARK "Build the GoogleBenchmark librar
 
 include(Sourcemeta)
 
+if(PROJECT_IS_TOP_LEVEL)
+  sourcemeta_target_clang_format(SOURCES
+    src/*.h src/*.cc
+    benchmark/*.h benchmark/*.cc
+    test/*.h test/*.cc)
+  sourcemeta_find_clang_tidy() # Locates CLANG_TIDY program
+endif()
+
+# Add clang-tidy to the targets. Clang-tidy uses the .clang-tidy file for configuration.
+set(CMAKE_CXX_CLANG_TIDY "${CLANG_TIDY_BIN}" "--allow-no-checks")
```

Comment:
Can you instead set this variable inside the clang tidy target?

Reply:
Thought of using global scope